pkg/tfgen/docs_test.go

```diff
@@ -197,7 +197,7 @@ func TestArgumentRegex(t *testing.T) {
 				"* `retention_policy` - (Required) A `retention_policy` block as documented below.",
 				"",
 				"---",
-				"* `retention_policy` supports the following:",
+				"`retention_policy` supports the following:",
```

Comment:
Verified via cs.github.com that whatever docs this code was intending to test the parsing of no longer exists in this format. That is, I was unable to find a nested block that starts with a bullet point and ends with "contains the following:".

It appears to be attempting to emulate a resource in Azure, but I also checked AWS, etc.